pymeasure/display/windows.py

```diff
@@ -557,360 +577,81 @@ class ManagedImageWindow(QtGui.QMainWindow):
 
 
     """
-
     EDITOR = 'gedit'
 
-    def __init__(self, procedure_class, x_axis, y_axis, z_axis=None, inputs=(), displays=(),
-                 log_channel='', log_level=logging.INFO, parent=None):
-        super().__init__(parent)
-        app = QtCore.QCoreApplication.instance()
-        app.aboutToQuit.connect(self.quit)
-        self.procedure_class = procedure_class
-        self.inputs = inputs
-        self.displays = displays
-        self.log = logging.getLogger(log_channel)
-        self.log_level = log_level
-        log.setLevel(log_level)
-        self.log.setLevel(log_level)
-        self.x_axis, self.y_axis, self.z_axis = x_axis, y_axis, z_axis
-        self._setup_ui()
-        self._layout()
-        self.setup_im_plot(self.im_plot)
-        self.setup_plot(self.plot)
-
-    def _setup_ui(self):
-        self.log_widget = LogWidget()
-        self.log.addHandler(self.log_widget.handler)  # needs to be in Qt context?
-        log.info("ManagedWindow connected to logging")
-
-        self.queue_button = QtGui.QPushButton('Queue', self)
-        self.queue_button.clicked.connect(self.queue)
-
-        self.abort_button = QtGui.QPushButton('Abort', self)
-        self.abort_button.setEnabled(False)
-        self.abort_button.clicked.connect(self.abort)
-
-        self.image_widget = ImageWidget(self.procedure_class.DATA_COLUMNS, self.x_axis, self.y_axis, self.z_axis)
-        self.plot_widget = PlotWidget(self.procedure_class.DATA_COLUMNS, self.x_axis, self.y_axis)
-        self.im_plot = self.image_widget.plot
+    def __init__(self, procedure_class, inputs=(), displays=(), x_axis=None, y_axis=None,
+                 log_channel='', log_level=logging.INFO, parent=None, sequencer=False,
+                 sequencer_inputs=None, sequence_file=None, inputs_in_scrollarea=False, directory_input=False, wdg_list=()):
+        self.x_axis = x_axis
+        self.y_axis = y_axis
+        self.log_widget = LogWidget("Experiment Log")
+        self.plot_widget = PlotWidget("Results Graph", procedure_class.DATA_COLUMNS, self.x_axis, self.y_axis)
         self.plot = self.plot_widget.plot
```

Comment:
I think that this line is not necessary anymore (as you removed all usage of `.plot` with the `widget_list`.

Reply:
I agree